draft/2022-02-09-this-week-in-rust.md

```diff
@@ -19,6 +19,7 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 ### Foundation
 
 ### Project/Tooling Updates
+* [This week in Fluvio #21: the programmable streaming platform](https://www.fluvio.io/news/this-week-in-fluvio-0021/)
```

Comment:
I'm getting "Page Not Found" when following this url

Reply:
A deploy got held up, but the link should be good now.